strawberry/experimental/pydantic/object_type.py

```diff
@@ -177,7 +178,7 @@ def wrap(cls: Any) -> Type[StrawberryTypeFromPydantic[PydanticModel]]:
         )
 
         wrapped = _wrap_dataclass(cls)
-        extra_strawberry_fields = _get_fields(wrapped, {})
+        extra_strawberry_fields = _get_fields(wrapped, TypeExtension(), {})
```

Comment:
Instantiating an empty type extension feels wrong to me.

Reply:
This I currently take as a temp code, I am planning on implementing `PydanticTypeExtension` once API is agreed upon.